### DISCUSSION TITLE
GraphQL Mutations Do Not Display Postgres Functions Created in Public Schema

### DISCUSSION THREAD

**Discussion by dittybijil** [supabase/supabase#18426] [Questions]
I have created some functions within the public schema of my project. However, I've encountered an issue where these functions are not appearing in the Supabase GraphQL mutation and query listings. I am seeking guidance on the steps required to include them in the mutation list. Your assistance in resolving this matter is greatly appreciated. Thank you in advance for your support

**Comment by GaryAustin1** [comment]
I don't believe that is supported based on:
https://github.com/orgs/supabase/discussions/7243
https://supabase.com/blog/graphql-now-available
![image](https://github.com/supabase/supabase/assets/54564956/74cedc84-fb69-4e62-ad16-cccbab92d8a1)
And no mention of adding it here: https://supabase.github.io/pg_graphql/changelog/

**Reply by dittybijil** [reply]
@GaryAustin1  Thank you for your response.  

The issue in question pertained to the function's return type. It is explicitly stipulated in the documentation that the system exclusively supports Built-in GraphQL scalar types, including Int, Float, String, and Boolean, alongside the facilitation of custom scalar types for function arguments and return types. The issue arose due to my utilization of 'set of' as a return type.
